packages/react-scripts/template/README.md

```diff
@@ -768,12 +768,24 @@ Install React Bootstrap and Bootstrap from npm. React Bootstrap does not include
 npm install --save react-bootstrap bootstrap@3
 ```
 
+For version 4:
```

Comment:
We can probably just replace the Bootstrap 3 instructions with Bootstrap 4. No need to include both.